src/backend.rs

```diff
@@ -260,14 +265,64 @@ fn pop_i32(ctx: &mut Context) -> GPR {
     gpr
 }
 
-pub fn add_i32(ctx: &mut Context) {
+pub fn i32_add(ctx: &mut Context) {
+    let op0 = pop_i32(ctx);
+    let op1 = pop_i32(ctx);
```

Comment:
There is a fair amount of redundancy between add/sub/etc. which we could factor out into a `binop` macro, though I'm ok either way, as there's not that much duplication. A `pop_i32_i32` which returns `(i32, i32)` might be handy here though.

Reply:
Already done in the current `HEAD`